src/http_crawler/__init__.py

```diff
@@ -19,7 +19,10 @@ def crawl(base_url, follow_external_links=True):
 
     while todo:
         url = todo.pop()
-        rsp = session.get(url)
+        rsp = session.get(url, allow_redirects=follow_redirects)
+
+        if rsp.status_code // 100 == 3:
+            continue
```

Comment:
How about `pause_at_redirects` or `scrap_redirects`?

Any thoughts on this?

> I think it would be a better approach to scrap the body of the 3xx HTTP response also, in addition to adding the url from location header.